### DISCUSSION TITLE
Request to Sentry API despite no Sentry integration

### DISCUSSION THREAD

**Discussion by ahollenbach** [vercel/community#921] [Help]
Hi! I noticed that my Vercel deployments are requesting `https://o370968.ingest.sentry.io/api/6260025/envelope/?sentry_key=<key>&sentry_version=7&sentry_client=sentry.javascript.react%2F7.13.0` - across multiple of my Next.js sites. Is Vercel injecting Sentry tracking?

* Neither of the two sites I set up are using Next.js analytics
* I do not have any sort of Sentry integration set up on my side via the Next.js console
* My code does not reference sentry anywhere

**Comment by samcx** [comment]
@ahollenbach I am unable to replicate this on a simple Next.js template. Is it possible to share with us this Vercel Deployment URL?
